index.html

```diff
@@ -34,6 +35,185 @@ <h1 class="text_title item-1">Vanessa Bonito</h1>
             </div>
             <div class="item-4"></div>
         </section>
+        <section class="grid-works">
+            <div class="div-title">
+                <h1 class="text_titleH2">Projects</h1>
+            </div>
+            <div class="container-work" id="project1">
+                <div class="img-work img-work-source1"></div>
+                <div class="div-title flex-work-title">
+                    <h1 class="text_titleH3">Project 1</h1>
+                </div>
+                <div class="flex-ul-work">
+                    <div class="ul-work-item">
+                        <ul class="ul-style">
+                            <li class="li-style">HTML/CSS</li>
+                            <li class="li-style">Ruby on rails</li>
+                            <li class="li-style">JavaScript</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="flex-button-work">
+                    <div class="button-work">
+                        See this project
+                    </div>
```

Comment:
Let's make this a button. You can read more about buttons [here](https://www.w3schools.com/tags/tag_button.asp). Same for the rest of the buttons in this section.